.travis.yml

```diff
@@ -1,15 +1,17 @@
 sudo: required
 env:
   # Run tests in parellel, up to 5 at once
-  - DEVICE=ios93 FINAL_GULP=coveralls
-  - DEVICE=ios93 TEST=e2e/driver
-  - DEVICE=ios93 TEST=e2e/testapp RECURSIVE=1
-  - DEVICE=ios93 TEST=e2e/uicatalog RECURSIVE=1
-  - DEVICE=ios93 TEST=e2e/safari RECURSIVE=1
-language: node_js
```

Comment:
Without the "language" line this defaults to Ruby, and we get the default node version.